microsoft365/outlook/contact/pkg/commands/createContact.go

```diff
@@ -0,0 +1,30 @@
+package commands
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/obot-platform/tools/microsoft365/outlook/contact/pkg/client"
+	"github.com/obot-platform/tools/microsoft365/outlook/contact/pkg/global"
+	"github.com/obot-platform/tools/microsoft365/outlook/contact/pkg/graph"
+	"github.com/obot-platform/tools/microsoft365/outlook/contact/pkg/util"
+)
+
+func CreateContact(ctx context.Context, givenName, surname, emails, businessPhones string) error {
```

Comment:
I think in this function we should check to make sure that the `givenName` and the `surname` are not empty, and return an error if they are.

Reply:
good point.

Reply:
actually these are not required. But I am going to check if there's at least one param provided.